packages/errors/test/test-ses0_18_3.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava';
+
+test('shim bare on ses 0.18.3', async t => {
+  // SES v0.18.3 predates the `bare` export.
+  await import('ses0_18_3');
```

Comment:
This is my first encounter with this dependency magic. Why does this need to be a dynamic import of the magic name rather than a static import? What happens if you just statically import it instead?

If you could statically import it, you could statically import the next one too, right?

Reply:
The initialization of `@endo/errors` throws an exception. I could move `ses` to a static import but `@endo/errors` has to be dynamic for the error to be catchable in the test.

Reply:
Makes sense. Resolving.